### PR TITLE
My work except Platform generation

### PR DESCRIPTION
**Pull request by SaveQaQ**
- Delete some absorbed scripts and game objects
- UI, we have a play button, tutorial button, death mode button, and endless mode button
- random obstacles implementation is done
- able to see the lumps change

### CODE REVIEW

**Comment by fardeenfs** [comment]
In the future, please split these issues and open PR for these issues separately.

For now, is this complete and ready to be merged?

**Comment by SaveQaQ** [comment]
> In the future, please split these issues and open PR for these issues separately.
> 
> For now, is this complete and ready to be merged?

**Comment by SaveQaQ** [comment]
> > In the future, please split these issues and open PR for these issues separately.
> > For now, is this complete and ready to be merged?

Yes, it should be completed and ready to be merged.

File: Assets/Scripts/PlatformMovement.cs
Comment (fardeenfs) [inline Assets/Scripts/PlatformMovement.cs:1]: Potential Merge Conflict with #22

**Review by fardeenfs — CHANGES_REQUESTED** [review]

**Review by fardeenfs — APPROVED** [review]

**Pull request merged**